apps/cloud-player/app.js

```diff
@@ -1,150 +1,81 @@
 var Application = require('@yodaos/application').Application
-var AudioFocus = require('@yodaos/application').AudioFocus
-var speechSynthesis = require('@yodaos/speech-synthesis').speechSynthesis
-var MediaPlayer = require('@yoda/multimedia').MediaPlayer
 var flora = require('@yoda/flora')
 var _ = require('@yoda/util')._
 var logger = require('logger')('cloud-player')
 
-var FloraUri = 'unix:/var/run/flora.sock#cloud-player'
-var GetStreamChannel = 'yodaos.apps.cloud-player.get-stream-channel'
-var MultimediaStatusChannel = 'yodaos.apps.multimedia.playback-status'
-var TtsStatusChannel = 'yodaos.apps.cloud-player.tts.status'
-
-var StatusCode = {
-  start: 0,
-  end: 1,
-  cancel: 2,
-  error: 3
-}
+var constant = require('./constant')
+var FloraUri = constant.FloraUri
 
 var app = Application({
   created: function created () {
+    this.voices = []
```

Comment:
Could this be as a map? It simplifies the `getVoice()`?

Reply:
Normally the voices array contains two item at most (one TRANSIENT and one DEFAULT). So no need to use a map here.